### ISSUE TITLE
Fix for using the python file to run on entire directory and produce json out

### ISSUE THREAD

**Issue by jogardi** [dougpuob/cppnamelint#67]
Python file is using jsonfile when it should should use jsonout.
```
diff --git a/script/cppnamelint.py b/script/cppnamelint.py
index 6128c7d..fa16c2c 100755
--- a/script/cppnamelint.py
+++ b/script/cppnamelint.py
@@ -340,7 +340,7 @@ def run_util_checkdir_files(exec_file_path, py_args, paired_samples:[], print_ou

         args_list: [] = ['check', paired['src'], '-config='+paired['cfg']]
         if py_args.json:
-            args_list.append('-jsonfile=' + py_args.json)
+            args_list.append('-jsonout=' + py_args.json)

         if py_args.inc:
             for file in py_args.inc:
```

P.S. Would be nice to check if jq is installed and run it on the json output if it is.

**Comment by dougpuob** [comment]
Hi @jogardi :
Thank you for your report.

**Comment by dougpuob** [comment]
Hi @jogardi : 
This issue was fixed in the #69 PR. Next time file a PR is welcome too.
